### ISSUE TITLE
Services names containing "@" show no provider icon in Dashboard

### ISSUE THREAD

**Issue by xsrf** [containous/traefik#5733]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

I defined a service with "@" in its name by a label in a docker swarm like this:
`traefik.http.services.api@internal.loadbalancer.server.port=1337`
Then I opened traefiks dashboard in the browser and looked for the service.

### What did you expect to see?

A blue docker icon next to the service indicating "Docker" as provider.

### What did you see instead?

No icon, the icon image failed to load.

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      2.0.2
Codename:     montdor
Go version:   go1.13.1
Built:        2019-10-09T19:26:05Z
OS/Arch:      linux/amd64
```

All service names generated from the docker provider are suffixed with "@docker". The provider icon filename is derived from the service name by the part after the "@", resulting in `dashboard/statics/providers/docker.svg` for a regular service provided by Docker.

If the service name contains an "@", like `api@internal`, the filename is wrong: `dashboard/statics/providers/internal@docker.svg`

To get the right provider icon filename, the service name should be cut after the last "@", not the first "@".

**Comment by ldez** [comment]
Hello,

`traefik.http.services.<name>.loadbalancer.server.port`

the character `@` is not authorized in service definition (`<name>`).

the character `@` can only be used to reference a service: `traefik.http.routers.myrouter.service=api@internal`

**Comment by xsrf** [comment]
Thank you for pointing that out and adding it to the docs.
Except for the failed Icon in the Dashboard, I found no problems with service names including "@", so I thought it would be an error of the Dashboard.

**Comment by traefiker** [comment]
Closed by #5734.
